scripts/reana

```diff
@@ -104,6 +106,91 @@ down () {
     kubectl delete -Rf configuration-manifests
 }
 
+demo () {
+    # Used to run REANA demo workflows.
+    # For now, hardcoded to run only helloworld-workflow.
+    demonstration=$2
+
+    # The code to check for flags is from http://stackoverflow.com/q/2875424
+    # Not sure if this is a good way or not.
+    flag_verbose=$(echo "$*" | grep -e "--verbose" -o)
+    flag_devel=$(echo "$*" | grep -e "--devel" -o)
```

Comment:
If I cahnge this to `flag_devel=$(echo $* | grep -e "--devel" -o)`, shellcheck complains:
* SC2048: Use "$@" (with quotes) to prevent whitespace problems.
* SC2086: Double quote to prevent globbing and word splitting.

With `flag_devel=$(echo $@ | grep -e "--devel" -o)` shellcheck complains:
* SC2068: Double quote array expansions, otherwise they're like $* and break on spaces.

If I use 'flag_devel=$(echo "$@" | grep -e "--devel" -o)' shellcheck doesn't complain.

I understand your point though.

I don't know enough about shell scripting to be able to say if any of these is good.

Should we change from `sh` to `bash` to use `if [[ "$@" != *"--devel"* ]]; then` if that would be a better alternative?